packages/smooth_app/lib/themes/constant_icons.dart

```diff
@@ -4,13 +4,22 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
 class ConstantIcons {
+  @visibleForTesting
+  const ConstantIcons();
+
+  static ConstantIcons get instance => _instance ??= const ConstantIcons();
+  static ConstantIcons? _instance;
+
+  /// Setter that allows tests to override the singleton instance.
+  @visibleForTesting
+  static set instance(ConstantIcons testInstance) => _instance = testInstance;
+
   static bool _isApple() =>
```

Comment:
`static` or not?

Reply:
It's private, so it doesn't really matter.  Since it doesn't depend on instance data, there's no reason to make it non-static.